src/llama_recipes/data/llama_guard/README.md

```diff
@@ -0,0 +1,98 @@
+# Finetuning Data Formatter
+
+The finetuning_data_formatter script provides classes and methods for formatting training data for finetuning a language model on a specific task. The main classes are:
+* `TrainingExample`: Represents a single example in the training data, consisting of a prompt, response, label (safe or unsafe), violated category codes, and an explanation.
+* `Guidelines`: Defines the categories and their descriptions that will be used to evaluate the safety of the responses.
+* `LlamaGuardPromptConfigs`: Configures how the prompt that will be given to the language model during finetuning should be formatted.
+* `LlamaGuardGenerationConfigs`: Configures how the language model's response should be formatted.
```

Comment:
Same

Reply:
I will omit this comment after this point, but in general recommend to replace all relevant instances of LM with LG in this doc :)